### ISSUE TITLE
New `wasmbin-derive` patch release?

### ISSUE THREAD

**Issue by fornwall** [RReverser/wasmbin#1]
Could you make a `wasmbin-derive` patch release, to get updated dependencies (would remove some duplications in the dependency tree for us)?

**Comment by RReverser** [comment]
Oh right, I updated the deps but postponed the release because I was waiting on https://github.com/panicbit/custom_debug/issues/18 to get rid of duplicate syn for downstream users.

Now that it's been fixed, I should've released both but it seems `cargo release` only picked up wasmbin itself.

**Comment by RReverser** [comment]
Done.

**Comment by fornwall** [comment]
Thanks!
